README.md

```diff
@@ -56,9 +56,16 @@ Generate an admin user:
 
     bundle exec rake db:devseed    
 
-Fire up a server and import some data.
+Fire up a server:
 
     foreman start -f Procfile.dev
+    
+Or, if you already have Elasticsearch and Redis running because the web and sidekiq actions timeout in `Procfile.dev`, you can start foreman like this:
+
+    foreman start -f Procfile_no_es_redis.dev
```

Comment:
Since this PR causes the Rails app to fetch info from ES during initialization, I suspect devs will have to start ES first before running foreman, i.e. we'll likely end up renaming Procfile_no_es_redis.dev to Procfile.dev.